docs/_release_description.md

```diff
@@ -1,6 +1,6 @@
 Install this version from pip with::
 
-    pip install tutor[full]==TUTOR_VERSION
+    pip install "tutor[full]==TUTOR_VERSION"
```

Comment:
From the commit message:
```
docs: wrap tutor[full] in quotes for zsh compatibility

The docs recommend commands like:

    pip install tutor[full]
    pip install -e ./tutor[full]

for installing Tutor. These work fine in bash. For zsh,
though, which is now the default on macOS, quotes are
needed, otherwise zsh will interpret the brackets as
special syntax:

    pip install "tutor[full]"
    pip install -e "./tutor[full]"

Caveat: I have not tested this myself since I don't
own a Mac, but I've read several issue reports to this
effect, such as:
https://github.com/pypa/pipenv/issues/2830#issuecomment-419593199
```

Reply:
I realized that I could install zsh myself and try this:
```zsh
$ pip install tutor[full]
zsh: no matches found: tutor[full]
$ pip install "tutor[full]"
Collecting tutor[full]
...
```
Single quotes also work, if you'd prefer those.

Reply:
Right. Sorry, I should have read the commit message.